frontend/src/app/features/quiz/question/question.component.css

```diff
@@ -0,0 +1,17 @@
+@keyframes fadeIn {
```

Comment:
add this to tailwind.config.js this will also be helpful to other

Reply:
Unused forgot to remove